packages/app/src/components/UIOverlay/Report/ReportSheet.tsx

```diff
@@ -89,7 +89,7 @@ export const ReportSheet = forwardRef((_props: PropsWithChildren<{}>, ref: Ref<R
     const onSubmit = async () => {
         if (!isValid) return
 
-        track({ name: 'report-submitted', timeTaken: Date.now() - openedAt.current! })
+        track({ name: 'Report Submitted', timeTaken: Date.now() - openedAt.current! })
```

Comment:
I think instead of using timeTaken you can use the Pirsch native "duration" param but you will have to check on that